packages/time/README.md

```diff
@@ -14,6 +14,11 @@ console.log(date.daysInMonth());
 const {semester} = require('@gradebook/time');
 console.log(semester.data.activeSemester);
 
+// Get semester in a readable format e.g. "Fall 2020"
+const {readableSemester} = require('@gradebook/time');
+semester = readableSemester('2020F');
+console.log(semester.season + ' ' + semester.year);
```

Comment:
Following the example on line 23

```suggestion
console.log(semester); // {season: 'Fall', year: 2020}
```

(please run the code and make sure the output is correct lol)

Reply:
We will need to redo the readme with the full class now part of time. How do you want that write up to look?

Reply:
Let's document the semester class with JSDoc comments for now (no need to update the README). We'll need to create a docs pipeline that extracts info from the code - so that the docs and code are in sync